redaxo/src/addons/be_style/scss/_icon.scss

```diff
@@ -292,6 +292,9 @@
 .rex-icon-view {
     @extend .fa-eye;
 }
+.rex-icon-hide {
```

Comment:
hier muss css noch kompiliert werden (ging auf meinem windows nicht -> class not found)

Reply:
hat thomas erledigt.